modules/miscutil/lib/obelixutils.py

```diff
@@ -112,24 +112,77 @@ def get_recommended_records(recid, user_id, collection="", threshold=70,
                                           'record_authors': ,
                                           }, ]
     """
-    from invenio.webstat import get_url_customevent
+    global _RECOMMENDATION_REDIS
+    from invenio.search_engine import check_user_can_view_record
+    recommendation_prefix = "reco_1::"
+
     if CFG_OBELIX_HOST == "":
-        # if not obelix:
+        # Obelix is not used
         return []
 
+    try:
+        if _RECOMMENDATION_REDIS is None:
+            import redis
+            _RECOMMENDATION_REDIS = redis.StrictRedis(host=CFG_OBELIX_HOST,
+                                                      port=6379,
+                                                      db=0)
+        # Get recommendation
+        key = "{0}recid::{1}".format(recommendation_prefix, recid)
+        recommendation = _RECOMMENDATION_REDIS.get(key)
+        if not recommendation:
+            return []
```

Comment:
Instead of returning no recommendations at all maybe you can call here `get_recommendation_obelix`